rpc/grpc/src/main/java/com/spotify/heroic/rpc/grpc/GrpcRpcProtocolServer.java

```diff
@@ -48,6 +48,7 @@
 import io.grpc.netty.NettyServerBuilder;
 import io.netty.channel.Channel;
 import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
```

Comment:
Do these changes pertain to the gRPC library upgrade?

Reply:
Yep. They were also needed to resolve a runtime exception too IIRC. Sergey came up with this solution.